### DISCUSSION TITLE
Is this abandoned?

### DISCUSSION THREAD

**Discussion by cpakken** [blinkdb-js/blinkdb#39] [Q&A]
Just stumbled across the library and it seems incredibly interesting. I wanted to find an in-memory store with indexable queries but all other libraries are very old (LokiJS, NeDB). But it seems the last update was 5 months ago, is this project abandoned? Is the current version stable enough to use in production?

**Comment by LarsFlieger** [comment]
Hey, same here. I'm looking for exactly something like this. I would love to work with millions of entries in an SPA (react). 
@maradotwebp Any updates on continuing this project?
